OracleUnifiedDirectory/dockerfiles/12.2.1.4.0/Dockerfile

```diff
@@ -0,0 +1,155 @@
+    echo "ldapmodify.java-args=-client -Dcom.sun.jndi.ldap.object.disableEndpointIdentification=true" >> ${ORACLE_HOME}/oud/config/java.properties && \
+    echo "ldapsearch.java-args=-client -Dcom.sun.jndi.ldap.object.disableEndpointIdentification=true" >> ${ORACLE_HOME}/oud/config/java.properties && \
+	echo "start-ds.java-args=-server -Xms256m -Dcom.sun.jndi.ldap.object.disableEndpointIdentification=true" >> ${ORACLE_HOME}/oud/config/java.properties && \
+    ${ORACLE_HOME}/oud/bin/dsjavaproperties
+
+#
+# Apply patch to OPatch
+# ---------------------
+RUN opatchzip=`ls ${PATCHES_BASE_DIR}/opatch_patch/p*.zip 2>/dev/null`; \
+    if [ ! -z "$opatchzip" ]; then \
+      cd ${PATCHES_BASE_DIR}/opatch_patch;  \
+      echo -e "\nBelow patch present in opatch_patch directory. Applying this patch:" ; \
+      ls p*.zip ; \
+      echo -e "" ; \
+      opatchfile=`ls p*.zip` ; \
+      $JAVA_HOME/bin/jar xf $opatchfile ; \
+      $JAVA_HOME/bin/java -jar ${PATCHES_BASE_DIR}/opatch_patch/6880880/opatch_generic.jar -silent oracle_home=$ORACLE_HOME; \
+      if [ $? -ne 0 ]; then \
+        echo "Applying patch to opatch Failed" ; \
+        exit 1 ; \
+      fi; \      
+      cd /tmp; \
+      rm  ${PATCHES_BASE_DIR}/opatch_patch/*.zip; \
+      rm -r ${PATCHES_BASE_DIR}/opatch_patch/; \
+    else \
+      echo -e "\nNo OPatch patch present in opatch_patch directory. Skipping patch application."; \
+    fi  
+
+#
+# Apply OUD Patches
+# -----------------
+RUN export OPATCH_NO_FUSER=TRUE && patchzips=`ls ${PATCHES_BASE_DIR}/patches/p*.zip 2>/dev/null`; \
+    if [ ! -z "$patchzips" ]; then \
+      cd ${PATCHES_BASE_DIR}/patches;  \
+      echo -e "\nBelow patches present in patches directory. Applying these patches:"; \
+      ls p*.zip; \
+      echo -e ""; \
+      for filename in `ls p*.zip`; do echo "Extracting patch: ${filename}"; $JAVA_HOME/bin/jar xf ${filename}; done; \
+      rm -f ${PATCHES_BASE_DIR}/patches/p*.zip; \
+      $ORACLE_HOME/OPatch/opatch napply -silent -oh $ORACLE_HOME -jre $JAVA_HOME -invPtrLoc ${BASE_DIR}/oraInst.loc -phBaseDir ${PATCHES_BASE_DIR}/patches; \
+      $ORACLE_HOME/OPatch/opatch util cleanup -silent; \
+      rm -rf ${PATCHES_BASE_DIR}/patches ${BASE_DIR}/oracle/cfgtoollogs/opatch/*; \
+      echo -e "\nPatches applied in OUD oracle home are:"; \
+      cd $ORACLE_HOME/OPatch; \
+      $ORACLE_HOME/OPatch/opatch lspatches; \
+    else \
+      echo -e "\nNo patches present in patches directory. Skipping patch application."; \
+    fi
+#
+
+# Final Image
+# -------------
+FROM base
+
+COPY --chown=oracle:oracle --from=builder $BASE_DIR $BASE_DIR
+COPY --chown=oracle:oracle --from=builder $JAVA_HOME $JAVA_HOME
+
+USER oracle
+WORKDIR ${ORACLE_HOME}
+  
+# run container health check
+HEALTHCHECK --interval=1m --start-period=5m \
+   CMD "${SCRIPT_DIR}/checkOUDInstance.sh" > /dev/null || exit 1
+   
+# Create and Start OUD Instance
+# ENTRYPOINT ["sh", "-c", "${SCRIPT_DIR}/createAndStartOUDInstance.sh"]
```

Comment:
If this is not required, please remove it instead of commenting it out.

Reply:
Removed.